README.md

```diff
@@ -23,13 +23,23 @@ Welcome to DocumentDB.
     $ pip install pydocumentdb
 
 
-2) To run tests:
+2) Testing:
+
+Some of the test files such as [crud_tests.py](https://github.com/Azure/azure-documentdb-python/blob/fe95945cae62ee4a68ae53bc5519bec11f07522c/test/crud_tests.py) require you to enter your Azure DocumentDB master key and host endpoint in that file: 
```

Comment:
Can you please get rid of the blob GUID from your hyperlink? That points to an old version of this file. Having the link as https://github.com/Azure/azure-documentdb-python/blob/master/test/crud_tests.py will always point to the latest version.